### PR TITLE
[5.3] Fix PHPDoc blocks for responses

### PR DESCRIPTION
**Pull request by rkgrep**
Use Symfony response where Illuminate response is not needed and 2 minor fixes

### CODE REVIEW

File: src/Illuminate/Foundation/Http/FormRequest.php [hunk 1]
```diff
@@ -156,7 +156,7 @@ public function response(array $errors)
     /**
      * Get the response for a forbidden operation.
      *
-     * @return \Illuminate\Http\Response
+     * @return  \Symfony\Component\HttpFoundation\Response
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Http/FormRequest.php]: Please remove the extra space.

File: src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php [hunk 2]
```diff
@@ -125,8 +125,8 @@ protected function tokensMatch($request)
      * Add the CSRF token to the response cookies.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @param  \Illuminate\Http\Response  $response
-     * @return \Illuminate\Http\Response
+     * @param  \Symfony\Component\HttpFoundation\Response  $response
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php]: why is this change needed? all other methods in this class don't do this

Comment (rkgrep) [inline src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php]: No other methods hint `Response`, but only `Request`.
The reason is that the method itself actually does not require any Illuminate-specific methods of Response
